src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -199,3 +199,16 @@ def upload_file_to_dsn(self, input_file, dataset_name, encoding=_ZOWE_FILES_DEFA
             response_json = self.write_to_dsn(dataset_name, file_contents)
         else:
             raise FileNotFound(input_file)
+
+    def delete_data_set(self, dataset_name, volume=None, member_name=None):
+        """Deletes a sequential or partitioned data."""
+        custom_args = self.__create_custom_request_arguments()        
```

Comment:
Hey @aadityasinha-dotcom, on the `next`  branch it's `__create_custom_request_arguments`

Reply:
Renamed this method to `_create_custom_request_arguments` with a single underscore in the `next` branch because the double underscore caused errors like the following:
`AttributeError: 'Files' object has no attribute '_Files__create_custom_request_arguments'. Did you mean: '_SdkApi__create_custom_request_arguments'?`

I'll update the name for this PR, just wanted to let you know for the future 🙂